Library/Homebrew/cmd/bundle.rb

```diff
@@ -61,42 +61,45 @@ class Bundle < AbstractCommand
 
           `brew bundle env`:
           Print the environment variables that would be set in a `brew bundle exec` environment.
+
+          If `--new-system` is passed or the environment variable `HOMEBREW_USE_NEW_SUBCOMMAND_SYSTEM` is set,
```

Comment:
Definitely don't expose this to users, please!

Reply:
removed